plugins/plugin-java/che-plugin-java-ext-lang-server/src/main/java/org/eclipse/che/plugin/java/server/rest/ClasspathService.java

```diff
@@ -19,35 +30,24 @@
 import org.eclipse.jdt.internal.core.JavaModel;
 import org.eclipse.jdt.internal.core.JavaModelManager;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.MediaType;
-import java.util.ArrayList;
-import java.util.List;
-
-import static java.util.Collections.emptyList;
-
 /**
  * Service for getting information about classpath.
  *
  * @author Valeriy Svydenko
  */
 @Path("java/classpath/")
-public class ClasspathService {
+public class ClasspathService implements ClasspathServiceInterface {
 
```

Comment:
The ` ClasspathService` component is now also used from the server-side class ` org.eclipse.che.plugin.testing.classpath.maven.server.MavenTestClasspathProvider`.

And when implementing the test for `MavenTestClasspathProvider`, mocking  `ClasspathService` directly brought classpath problems, while mocking the implemented interface works perfectly.

see https://github.com/davidfestal/che/blob/5a698ac9aed88f54c6eb98372db493ef39b3ec8b/plugins/plugin-testing-java/plugin-testing-classpath/che-plugin-testing-classpath-maven-server/src/test/java/org/eclipse/che/plugin/testing/classpath/maven/server/MavenTestClasspathProviderTest.java#L46